.jenkins/docker_cuda_theano/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM nvidia/cuda:8.0-cudnn5-devel-ubuntu16.04
```

Comment:
While at it, if you can update to cudnn v6, it would be great. We started to wrap it.